nano/core_test/websocket.cpp

```diff
@@ -124,6 +124,46 @@ TEST (websocket, confirmation)
 	ASSERT_TIMELY (5s, future.wait_for (0s) == std::future_status::ready);
 }
 
+// Tests getting notification of a started election
+TEST (websocket, started_election)
+{
+	nano::system system;
+	nano::node_config config (nano::get_available_port (), system.logging);
+	config.websocket_config.enabled = true;
+	config.websocket_config.port = nano::get_available_port ();
+	auto node1 (system.add_node (config));
```

Comment:
We should use the more standard initialisation syntax for new code,

auto node1 = system.add_node (config);